Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.h

```diff
@@ -99,13 +102,17 @@ namespace AzToolsFramework::ViewportUi::Internal
     private:
         void PrepareWidgetForViewportUi(QPointer<QWidget> widget);
 
+        // ViewportImGuiNotificationsBus overrides ...
+        void ImGuiActive(bool active) override;
```

Comment:
suggestion: Just thought of this now, but for notification buses, the functions usually start with `On`, so this could be improved by renaming to `OnImGuiActivated`